package.json

```diff
@@ -88,6 +88,8 @@
     "react": "17.0.1"
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.13.10",
+    "@babel/eslint-plugin": "^7.13.10",
```

Comment:
I think these should be `devDependencies` since they're for lint that runs when you're working on react-native itself? The dependencies for users of react-native come down as part of the `@react-native-community/eslint-config` install.